test/android/core/javatests/src/org/xwalk/core/xwview/test/ShouldOverrideUrlLoadingTest.java

```diff
@@ -469,7 +471,7 @@ public Boolean call() throws Exception {
             }
         });
 
-        assertEquals(1, mShouldOverrideUrlLoadingHelper.getCallCount());
+        assertEquals(shouldOverrideUrlLoadingCallCount, mShouldOverrideUrlLoadingHelper.getCallCount());
```

Comment:
yes, it's "0". I referred to WebView. I think this style means the test just compare the function call count before and after getting request from web server.